src/Handlers/PaymentResponseHandler.php

```diff
@@ -243,19 +243,24 @@ function (Context $context) use ($orderTransactionId, $customFields) {
 
```

Comment:
iDeal payments were returning with status `Pending` which was not handled here previously. Also, the exception was not logged